openmc/mgxs/library.py

```diff
@@ -524,6 +540,18 @@ def build_library(self):
                     mgxs.legendre_order = self.legendre_order
                     mgxs.histogram_bins = self.histogram_bins
 
+                if self.by_nuclide:
+                    try:
+                        domain_nuclides = domain.get_nuclides()
+                    except AttributeError:
+                        domain_nuclides = None
+                    if self.nuclides:
+                        if domain_nuclides:
+                            mgxs.nuclides = [nuclide for nuclide in self.nuclides if nuclide in domain_nuclides] + [
+                                "total"]
```

Comment:
This line is getting a little long:
```suggestion
                            mgxs.nuclides = [
                                nuclide for nuclide in self.nuclides
                                if nuclide in domain_nuclides
                            ] + ["total"]
```